plugins/inputs/ecs/types_test.go

```diff
@@ -42,6 +42,7 @@ func Test_mergeTaskStats(t *testing.T) {
 	mergeTaskStats(parsedMetadata, parsedStats)
 
 	for i := range parsedMetadata.Containers {
+		//nolint:testifylint // values are in proper order (actual, expected)
 		require.Equal(t, validStats[parsedMetadata.Containers[i].ID], &parsedMetadata.Containers[i].Stats)
```

Comment:
The values here do seem to be in the correct order so the lint suppression is correct, but the comment is written backwards

Reply:
Of course, my bad.